src/python/vdo-setup-dmtest/config.yaml

```diff
@@ -0,0 +1,28 @@
+# Devices to be used by dmtest-python
+DATA_DEVICE: /dev/mapper/scratch-vdo_scratch
+METADATA_DEVICE: /dev/mapper/scratch-vdo_scratch
```

Comment:
Are these OK to be the same device?

Reply:
We do not know yet. That is something I am trying to figure out.